### DISCUSSION TITLE
2022-09-28 Weekly Meeting 75 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#299] [Weekly Meetings]
Weekly Meeting 75 is happening on Wednesday September 28, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [OMI Website update](https://github.com/omigroup/website)
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [UX Research: TBD](https://github.com/omigroup/ux-research) (setting up the repo) 
  - #222 repo is formed, when do we have the first meeting? https://github.com/omigroup/ux-research/issues/6
- any others?

### Discussions: 
- Retro Results - Let's discuss!

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
- weak turnout for dev hour this week. May want to consider reducing the frequency or canceling for now.

**Comment by mrmetaverse** [comment]
# M3 HQ tour 
Started with a quick tour of M3's new HQ 

![image](https://user-images.githubusercontent.com/63426722/192829912-8f98bb09-a871-4b92-ba19-7a3dfd50cbfa.png)
![image](https://user-images.githubusercontent.com/63426722/192829964-4a07b8a8-5ae5-491b-93c6-bad627393539.png)
![image](https://user-images.githubusercontent.com/63426722/192829988-082b617f-5982-4a81-b3e6-fbc4694b7d6a.png)

Can use tools like unigltf and vrm to simplify import/export workflows 
![image](https://user-images.githubusercontent.com/63426722/192831033-e084f2e9-8057-4163-a277-c213fc5e4b0d.png)

Metafactory will go here on the 2nd floor: 
![image](https://user-images.githubusercontent.com/63426722/192830339-fc4954f9-0c7f-4004-aa31-8caab44249e3.png)


Proposed concept of adding an OMI space next door. Related to #218 

Evo to add design resources to [omigroup/omigroup/assets](https://github.com/omigroup/omigroup/tree/main/assets)

Would be nice to be able to change in one place and have it reflect in all platforms that it is in. 
![image](https://user-images.githubusercontent.com/63426722/192831269-d87b038e-12b4-40fb-9165-cd40e523646e.png)

**Comment by mrmetaverse** [comment]
# Media 
- seeking short interview clips 
- seeking editing for some of the clips 
- some new events coming up, thinking through where this content may fit into our broader media landscape. 
- meeting in 2 weeks to actively go through and curate on that front. October 11th, 2022 at 16:00 UTC 

If you are interested in the media and educational storytelling side of our work, we hope you will get involved!

**Comment by mrmetaverse** [comment]
# MSF Delegates 
- you can keep up with the latest [here](https://github.com/omigroup/MSF-Delegates/discussions)
- 3d assets domain group proposal was tentatively approved. 
- there is a meeting tonight for 3d assets AND wearables.

**Comment by madjin** [comment]
updates from M3 avatar interop:
- Discussed VRM 1.0 spec yesterday: https://vrm.dev/en/vrm1/changed.html
- Launched Webaverse MMO for improving docs / readmes: https://github.com/webaverse-mmo
  - Why: Webaverse solves a lot of hard challenges and is great for case studies but needs better documentation
  - Docs: https://hackmd.io/@xr/webaverse-mmo
- MetaRick is doing weekly Uncanny Alley Meetups on Tuesdays, hosted a workshop in hyperfy on how to make VRMs
![brave_UJZHCVBIs3](https://user-images.githubusercontent.com/32600939/192838652-fd25a707-0202-4889-8eea-25b0a9ec481d.png)

**Comment by mrmetaverse** [comment]
# UX 
- looking for the best time to meet to kickoff

**Comment by mrmetaverse** [comment]
# Retro 
We are continuing our work [on this retrospective](https://github.com/orgs/omigroup/projects/10). 

We will run a new backlog refinement once finished.
